src/components/popover/bl-popover.css

```diff
@@ -0,0 +1,31 @@
+.popover {
+  --arrow-display: var(--bl-popover-arrow-display, none);
+  --background-color: var(--bl-popover-background-color, var(--bl-color-primary-background));
+  --border-color: var(--bl-popover-border-color, var(--bl-color-primary-hover));
+  --padding: var(--bl-popover-padding, var(--bl-size-m));
+  --border-radius: var(--bl-popover-border-radius,  var(--bl-size-3xs));
+  --position: var(--bl-popover-position, absolute);
```

Comment:
Default value should be `fixed`, right?

Reply:
If I use fixed there are some issues e.g scrolling with screen etc. I think if we use absolute it is ok. Do we have any problem with absolute in another components?

Reply:
OK, lets fix other issues, maybe we can try out with different values to understand the difference.